contrib/forms/handwritten-notes/hw-notes.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Handwritten Notes Form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Tyler Wrenn <tyler@tylerwrenn.com>
+ * @copyright Copyright (c) 2020 Tyler Wrenn <tyler@tylerwrenn.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ **/
+
+require_once("../../globals.php");
+require_once("$srcdir/classes/Document.class.php");
```

Comment:
Above line is not needed. the class autoloader in globals.php makes Document class already available.